package.json

```diff
@@ -35,8 +35,10 @@
     "@grpc/grpc-js": "^1.2.11",
     "@grpc/proto-loader": "^0.5.6",
     "@profusion/apollo-validation-directives": "^2.1.3",
+    "apollo-server-errors": "^2.5.0",
     "apollo-server-express": "^2.21.2",
     "axios": "^0.21.1",
+    "bip66": "^1.1.5",
```

Comment:
this was a missing dependency provided by another package?

Reply:
Yep